index.html

```diff
@@ -531,6 +532,7 @@ <h2 class="form-text-primary">
           placeholder="Email address"
           required
         />
+        <small id="errorMsg"></small>
```

Comment:
- [ ] Great work on your portfolio, Kindly notice that the error message should show up close to the "submit" button as required, you are showing it close to email input.